packages/suite/src/components/onboarding/DeviceAnimation/index.tsx

```diff
@@ -80,14 +81,16 @@ export const DeviceAnimation = ({
     const hologramRef = useRef<HTMLVideoElement>(null);
 
     // if no Trezor available, show flagship model
-    const deviceModel = (getDeviceModel(device) || DeviceModel.TT).toLowerCase();
+    const deviceModelInternal = (
+        device?.features?.internal_model || DeviceModelInternal.T2T1
+    ).toLowerCase();
 
     // T1 bootloader before firmware version 1.8.0 can only be invoked by holding both buttons
     const deviceFwVersion = device?.features ? getFirmwareVersion(device) : '';
     let animationType = type;
     if (
         type === 'BOOTLOADER' &&
-        deviceModel === DeviceModel.T1 &&
+        deviceModelInternal === DeviceModelInternal.T1B1 &&
```

Comment:
`deviceModelInternal` is lower cased here

Reply:
Nice catch, so it was not working even before right?

Reply:
80aa9b34a853ba6c8b9e818b3543f767b05b6f19